package.json

```diff
@@ -189,7 +189,7 @@
     "exitzero": "^1.0.1",
     "jest": "^29.5.0",
     "npm-run-all": "^4.1.5",
-    "nx": "14.8.6",
+    "nx": "15.0.0",
```

Comment:
q: Any reason that we only went to base 15? Not the latest minor version of 15?

Reply:
Yes. Some minor version (4 or 5) has broke build and dev. I think we need to update elasticui. Ill make new pr to update it to latest minor after some digging

Reply:
Or, maybe that was my mistake. I tried to update both react router, elastic ui and nx at the same time. Only after few hours i noticed that i downgraded router instead of updating. Anyway, i will make new pr very soon